apps/condo/domains/contact/utils/clientCard.tsx

```diff
@@ -0,0 +1,145 @@
+            clientPhone_contains: $phone,
+            isResidentTicket: false,
+            clientPhone_not: null,
+            clientName_not: null,
+        }, first: 10) {
+          id
+          property { ${TICKET_PROPERTY_FIELDS} }
+          clientName
+          clientPhone
+          unitName
+      }
+      
+        employees: allOrganizationEmployees(
+          where: {
+            organization: { id: $organizationId },
+            phone_contains: $phone,
+          }, first: 10) {
+            id
+            name
+            phone
+        }
+    }
+`
+
+async function _search (client, query, variables) {
+    return await client.query({
+        query: query,
+        variables: variables,
+        fetchPolicy: 'network-only',
+    })
+}
+
+export function searchByPhone (organizationId) {
+    if (!organizationId) return
+
+    return async function (client, phone) {
+        const { data, error } = await _search(client, SEARCH_BY_PHONE, { phone, organizationId })
+
+        if (error) console.warn(error)
+
+        const employees = []
+        const contacts = data.contacts.map(({ id, phone, property, unitName }) => ({
+            value: id, phone, property, unitName, type: ClientType.Resident,
+        }))
+        const tickets = data.tickets.filter(ticket => {
+            const employee = data.employees.find(employee => employee.phone === ticket.clientPhone)
+            if (employee) {
+                employees.push({
+                    value: ticket.id,
+                    phone: ticket.clientPhone,
+                    property: ticket.property,
+                    unitName: ticket.unitName,
+                    type: ClientType.NotResident,
+                    isEmployee: true,
+                })
+                return false
+            } else return true
+        }).map(ticket => ({
+            value: ticket.id,
+            phone: ticket.clientPhone,
+            property: ticket.property,
+            unitName: ticket.unitName,
+            type: ClientType.NotResident,
+        }))
+
+        return [...contacts, ...tickets, ...employees]
+    }
+}
+
+const SELECT_OPTION_ROW_GUTTER: [Gutter, Gutter] = [120, 0]
+const LINK_STYLES = { fontSize: fontSizes.label, color: colors.black }
+export const mapToSelectOption = ({ id, phone, property, unitName, type, DeletedMessage }) => (
+    <Select.Option key={id} value={id} title={phone}>
+        <Typography.Link href={`/phone/${phone}?tab=${getClientCardTabKey(property.id, type, unitName)}`} style={LINK_STYLES}>
+            <Row gutter={SELECT_OPTION_ROW_GUTTER}>
+                <Col>
+                    {phone}
+                </Col>
+                <Col>
+                    {property ? getAddressRender(property) : DeletedMessage}
+                </Col>
+            </Row>
+        </Typography.Link>
+    </Select.Option>
+)
+
+export const getClientCardTabKey = (propertyId: string, type: ClientType, unitName: string): string =>
```

Comment:
If `unitName` is not required, you can mark it appropriately
```suggestion
export const getClientCardTabKey = (propertyId: string, type: ClientType, unitName: string?): string =>
```